### PR TITLE
[fix](ip) change IPv6 to little-endian byte order storage (like IPv4)

### PR DESCRIPTION
**Pull request by sjyango**
## Proposed changes

Issue Number: close #xxx

change IPv6 to little-endian byte order storage (like IPv4), in order to ensure unified storage and computation.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by sjyango** [comment]
run buildall

**Comment by sjyango** [comment]
run buildall

File: be/src/vec/common/format_ip.h [hunk 1]
```diff
@@ -194,15 +194,135 @@ inline bool parseIPv4whole(const char* src, unsigned char* dst) {
     return end != nullptr && *end == '\0';
 }
 
+/// integer logarithm, return ceil(log(value, base)) (the smallest integer greater or equal than log(value, base)
+inline constexpr UInt32 intLog(const UInt32 value, const UInt32 base, const bool carry) {
```
Comment (HappenLee) [inline be/src/vec/common/format_ip.h:198]: remove the unless code inline

**Comment by sjyango** [comment]
run buildall

**Comment by sjyango** [comment]
run buildall

**Comment by sjyango** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by zhangstar333 — APPROVED** [review]

**Pull request merged**